tool/data/config/logs/eventConfig.go

```diff
@@ -25,6 +26,9 @@ func (config *EventConfig) ToMap(ctx *runtime.Context) (string, map[string]inter
 	}
 	resultMap["log_group_name"] = config.LogGroup
 	resultMap["log_stream_name"] = config.LogStream
+	if config.LogGroupClass != "" {
+		resultMap["log_group_class"] = config.LogGroupClass
```

Comment:
is there any validation on the log_group_class configuration? ie. check for STANDARD, INFREQUENT_ACCESS, etc?

Reply:
There are validations to make sure that the only STANDARD and INFREQUENT_ACCESS are accepted https://github.com/aws/amazon-cloudwatch-agent/pull/968/files#diff-168367fbe6308e4f71d0d767b73db774d4a04414627718695c13c7d5709b2e05R57